src/Dfe.PlanTech.Application/Submissions/Queries/GetSubmissionStatusesQuery.cs

```diff
@@ -56,17 +56,15 @@ public async Task<List<SectionStatusDto>> GetSectionSubmissionStatuses(string ca
     /// <param name="completed"></param>
     /// <returns></returns>
     private static IQueryable<SectionStatusNew> GetLatestSubmissionStatus(IQueryable<Submission> submissionStatuses, bool completed)
-    => submissionStatuses.Select(submission => new SectionStatusNew()
-    {
-        DateCreated = submission.DateCreated,
-        Completed = submission.Completed,
-        Maturity = submission.Maturity,
-        SectionId = submission.SectionId,
-        Status = submission.Completed ? Status.Completed : submission.Responses.Count != 0 ? Status.InProgress : Status.NotStarted,
-    })
-    .GroupBy(submission => submission.SectionId)
-    .Select(grouping => grouping
-        .OrderByDescending(status => completed && status.Completed)
-        .ThenByDescending(status => status.DateCreated)
-        .First());
-}
+        => submissionStatuses.Select(submission => new SectionStatusNew()
+        {
+            DateCreated = submission.DateCreated,
+            Completed = submission.Completed,
+            Maturity = submission.Maturity,
+            SectionId = submission.SectionId,
+            Status = submission.Completed ? Status.Completed : submission.Responses.Count != 0 ? Status.InProgress : Status.NotStarted,
+        })
+        .Where(submission => !completed || submission.Completed)
+        .GroupBy(submission => submission.SectionId)
+        .Select(grouping => grouping.OrderByDescending(status => status.DateCreated).First());
```

Comment:
I don't think this is exactly the same (correct me if I'm wrong)
but the former orders by completion, so favours the completed submissions, but would give a non-complete one if there were no completed submissions, 
the new one filters out incomplete ones so you might get nothing

Reply:
I _think_ I've put it back to how it was, because I think you're right. Could you double check it for me please?

Reply:
I am content that this is reverted 👍